src/main/java/crazypants/enderio/config/Config.java

```diff
@@ -1269,6 +1305,37 @@ public static void processConfig(Configuration config) {
                         "Number of blocks teleported when no block is being looked at.")
                 .getInt(teleportStaffFailedBlinkDistance);
 
+        teleportStaffOriginalControls = config.get(
+                sectionPersonal.name,
```

Comment:
Hmm, I was originally thinking that it should be placed in Personal Settings because it looked like Staff Settings was all server-side-affecting configs (e.g. things like max range and energy consumption), whereas this one was purely client-side. But it looks like there's some other client-side configs in there too, like render size, so I moved the new configs. Thanks!